### DISCUSSION TITLE
New UI: super SLOW, can't opt out (global navigation update)

### DISCUSSION THREAD

**Discussion by bryango** [community/community#70277] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**Update:** I learn that this was beta tested as the [**Global navigation update**](https://github.com/orgs/community/discussions/52083).

Today I am greeted with a new github UI that I can no longer opt out (at least not in the "feature preview"). I actually quite enjoy the new aesthetics, but it's **super slow** under bad network connections; see below for a screenshot of my sidebar:

![github-new-ui](https://github.com/community/community/assets/26322692/6d26a2cc-c6f8-4fd0-82e8-a8d31bf72793)

Please fix this or provide an option to opt out. Thank you very much!

> **Note:** there are existing threads about the new UI but I would like to emphasize that it's slow and I can't opt out.

**Comment by SayaAEP** [comment]
![image](https://github.com/community/community/assets/91467886/d093ab61-203f-4ca6-83fa-dd79163aee83)
new navigation ui is suck.

**Comment by Vallek** [comment]
Yeah fetching same menu items on every page is typical 2023 web. There's zero reasons why it shouldn't be static. Can we have the old ui back please.
